config/pmd/ruleset.xml

```diff
@@ -21,13 +21,12 @@
   <description>
     Spark Operator Ruleset
   </description>
-  <rule ref="category/java/bestpractices.xml"/>
+  <rule ref="category/java/bestpractices.xml">
+    <exclude name="JUnitAssertionsShouldIncludeMessage" />
```

Comment:
This is recommended, but we have many assertions already. We don't need to enforce it.